reference/payments/process_payments.yml

```diff
@@ -80,6 +80,8 @@ paths:
                         - $ref: '#/components/schemas/StoredPayPalAccount'
                         - $ref: '#/components/schemas/GiftCertificate'
                         - $ref: '#/components/schemas/StoreCredit'
+                        - $ref: '#/components/schemas/CreditCard' 
```

Comment:
I don't believe we need `CreditCard`, it is already covered by `Card` component.
Both the examples seem to be related to tokenized cards, one with the iin and expiration date, and other without them.
I am going to loop in @nkostohryz or @bc-alextop to fill in the correct names here.

Reply:
@rabinshr @bc-traciporter Yes, seems that both examples is TokenizedCard, one with required only field(token) and other with all optional fields.

Reply:
@bc-traciporter yes